torch_geometric/nn/conv/hgt_conv.py

```diff
@@ -187,7 +305,6 @@ def forward(
     def message(self, k_j: Tensor, q_i: Tensor, v_j: Tensor, rel: Tensor,
                 index: Tensor, ptr: Optional[Tensor],
                 size_i: Optional[int]) -> Tensor:
-
```

Comment:
hmmm im not 100% sure, @rusty1s can u advise on this? i did my best to match up the shapes/dimensions and have the hgt CI tests passes.

Reply:
I think the two implementations are definitely different, but I believe we are fixing the original implementation here (since `HGTConv` is indeed designed to share message passing over all edge types AFAIK).